### PR TITLE
Add support for Nielsen Analytics

### PR DESCRIPTION
**Pull request by uniquelygeneric**
In reference to ampproject/amphtml issue [#3337](https://github.com/ampproject/amphtml/issues/3337).

### CODE REVIEW

**Comment by cramforce** [comment]
@avimehta @rudygalfi Could you review? Thanks!

File: extensions/amp-analytics/0.1/vendors.js [hunk 1]
```diff
@@ -418,6 +418,27 @@ export const ANALYTICS_CONFIG = {
       'image': true,
     },
   },
+
```
Comment (avimehta) [inline extensions/amp-analytics/0.1/vendors.js]: remove trailing spaces here.

File: extensions/amp-analytics/0.1/vendors.js [hunk 2]
```diff
@@ -418,6 +418,27 @@ export const ANALYTICS_CONFIG = {
...
     },
   },
+  
+  'nielsen': {
+    'vars': {
+      'appid': '',
+      'assetName': '',
+      'segA': 'NA',
+      'segB': 'NA',
+      'segC': 'NA'
```
Comment (avimehta) [inline extensions/amp-analytics/0.1/vendors.js]: add trailing commas here and below.

**Comment by avimehta** [comment]
lgtm. just the few lint errors that need to be fixed.

**Comment by avimehta** [comment]
cool. @uniquelygeneric  Any updates on CLA?

**Comment by uniquelygeneric** [comment]
Our Legal council is currently reviewing the CLA, I hope to have it completed soon.

**Comment by avimehta** [comment]
@uniquelygeneric any updates here?

**Comment by uniquelygeneric** [comment]
Legal is still trying to get the proper sign-off within Engineering.
Sorry for the delay. I will update you once it's completed.

**Comment by uniquelygeneric** [comment]
We are still waiting on legal approval (my apologies). However, if #2487 is implemented, we will update our implementation outlined in this issue to use a different protocol. This newer protocol will hopefully increase our speed pushing this through legal.

**Comment by rudygalfi** [comment]
Hi @uniquelygeneric -- any updates on your end?

By the way, #2487 has been completed and should land in this week's production build.

**Comment by uniquelygeneric** [comment]
Hey @rudygalfi, we are currently planning on making some changes to our backend to accommodate the changes implemented in #2487. This is due to the lack of persistence in vars used across triggers. We expect this to be a relatively minor fix, and I will update our spec once they have been completed.

**Comment by stephanfowler** [comment]
Hello @uniquelygeneric. We (the Guardian) are really keen to see support for Nielsen tracking; is there an update on progress with this?

**Comment by uniquelygeneric** [comment]
Hey, @stephanfowler, we still have to make some changes to our backend to support our measurement of AMP. Do you have an email address we can reach out to you with?

**Comment by stephanfowler** [comment]
@uniquelygeneric thank you, that would be great, I'm stephan.fowler@theguardian.com

**Comment by jooj123** [comment]
@uniquelygeneric any update on this one - we would love for this one to get in

**Comment by uniquelygeneric** [comment]
@jooj123, sorry for the delayed response. I was away for the holidays. we are in the process of making some changes on our backend and hope to have AMP support by the end of this quarter

**Comment by adelinamart** [comment]
@uniquelygeneric @rudygalfi @lannka what are we missing here? Thanks.

**Comment by uniquelygeneric** [comment]
@adelinamart Nielsen is currently testing a solution. Once it passes our tests we will submit to Google for inclusion in the AMP spec.

**Comment by uniquelygeneric** [comment]
Closing this pull request. Will be performing a new one off the new spec

**Comment by jooj123** [comment]
@uniquelygeneric whats the hold up - anything we can help with ? 
This would now be the 3rd attempted PR so we are a bit concerned at this point and may need to look at alternatives

We have been waiting for this one since the middle of last year

**Comment by uniquelygeneric** [comment]
@jooj123 We have submitted our spec at #9356. At the moment the only holdup is that we are facing issues with getting the Google CLA signed, but we hope to have that resolved soon.

**Pull request closed without merging**